edb/server/protocol/ai_ext.py

```diff
@@ -324,6 +324,19 @@ async def _generate_embeddings(
     raise RuntimeError(f"unsupported model provider: {provider.name}")
 
 
+async def _start_chat(
```

Comment:
RAG API docs need to be reflected somewhere more than a commit message

Reply:
For sure.  The commit message is a starting point.